app/static/css/Cloudbreak.min.css

```diff
@@ -2196,6 +2196,158 @@ textarea.input-lg, textarea.form-group-lg .form-control, select[multiple].input-
     background-color: #fff
 }
 
+.popover {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1060;
+    display: none;
+    min-width: 200px;
```

Comment:
please do not override the min-width property of the main popover class because every new popover will have the same width. You can use the popover-class attribute to append a custom css class to the popover like:
```
<i class="fa fa-question-circle" popover-class="my-large-popover" popover="text"  popover-trigger="mouseenter" />
```